Extensions/retags.js

```diff
@@ -27,7 +27,7 @@ XKit.extensions.retags = {
 		var inner, $el;
 		ms.forEach(function(m){
 			$el = $(m.addedNodes);
-			inner = $el.find('.note:not(.ui_post_badge)');
```

Comment:
No -- activity page still works. (If I'm understanding correctly what's happening here, this is only for the mutation observer detecting the note popover.)